packages/overreact/src/hooks/use-mutation.js

```diff
@@ -155,8 +158,10 @@ export function useMutation(dataRefId, spec, config) {
         deleteRecords(store, requestContract, recordsData);
       }
 
+      const data = getRawData(processedResponse);

```

Comment:
if this is urgent fix for our replacement, we can check in this part.
another possible fix is we don't include this id in processedResponse, so all hooks are fixed.
the way we achieve this is by fixing the part when we add this id. instead of adding to the raw object of response, we add it as metadata of the response:
``` javascript
storeItem = {
   overreactId: xxx,
   raw: responseData,
}
```

the benefit is we don't have to change the reference of this response object, and it will auto apply to all hooks.

Reply:
Echo Yao